build/tutorials/tutorials-contest/pangolin-token-subgraph/README.md

```diff
@@ -0,0 +1,480 @@
+
+Below is a diagram that further illustrates the points discussed.
+
+
+
+![alt_text](images/image2.png "Avalanche Architecture")
+
+
+Source: [https://docs.avax.network/learn/platform-overview](https://docs.avax.network/learn/platform-overview)
+
+Avalanche enables the creation of both private and public blockchains. Custom blockchains can be created using the Platform Chain and requirements can be set for validators joining the subnet that validates transactions on that custom blockchain in line with the business needs of an enterprise or entity. Assets can also be swapped in a cross-chain fashion between the different chains in the Avalanche network, for example, assets can be transferred from the Exchange Chain to the Contract Chain and vice versa. For more explanations about concepts in Avalanche, you can consult the official [documentation](https://docs.avax.network/).
+
+### The Graph and Avalanche
+
+In line with The Graph’s vision of providing open access to the world’s data and Avalanche’s goal of being an open, programmable, smart contracts platform that powers the next wave of scalable decentralized applications, both entities [announced a partnership](https://medium.com/avalancheavax/avalanche-integrates-the-graph-to-bring-its-querying-and-indexing-to-avalanche-536e48043486) that allows on-chain data from Avalanche to be indexed and queried through The Graph protocol. This means that Web3 developers can now access Avalanche data through subgraphs and build data-intensive applications while also having access to data from other blockchains supported by The Graph protocol.
+
+### Interacting with Avalanche Data via the Graph: Pangolin Example
+
+In this section, you will learn how to query Avalanche data from an already deployed subgraph on The Graph’s [hosted service](https://thegraph.com/legacy-explorer/). The subgraph you will use indexes data from the [Pangolin](https://pangolin.exchange/) decentralized exchange on Avalanche. Pangolin is an [Automated Market Maker (AMM)](https://support.avax.network/en/articles/4840276-what-is-an-automated-market-maker-amm) on Avalanche, similar in operation to [Uniswap](https://uniswap.org/). It can be used to swap Ethereum and Avalanche assets and has fast settlement times and low transaction fees. The Graph’s hosted service provides a playground for deployed subgraphs. The playground is an IDE-like environment based on [GraphiQL](https://github.com/graphql/graphiql) where you can write sample queries to fetch data supported by a subgraph. It is a valuable tool when getting to know the queries that are supported by third-party subgraphs deployed by various projects. The Pangolin exchange subgraph can be accessed [here](https://thegraph.com/legacy-explorer/subgraph/dasconnor/pangolin-dex?selected=playground). Below is an image of the playground interface.
+
+
+![alt_text](images/image3.png "The Graph Playground")
```

Comment:
Rename images and paths

Reply:
Images names have been changed to the naming convention.